_posts/2018-04-15-customreaders.md

```diff
@@ -0,0 +1,223 @@
+
+## Writing Custom Readers and Parsers for MDAnalysis
+
+
+molecular simulation full of custom formats, or variants on existing formats
+
+these can be tricky to read as columns get rearranged
+
+similarly, the number of fields used can vary greatly between different formats
+
+in this blog, we show how MDAnalysis can be used to read any format,
+including adding custom attributes
+
+
+```python
+myfile = """\
+7
+My custom file thingy
+C red      1.0 1.0 1.0
+H blue     1.0 1.5 1.5
+H blue     1.0 0.5 0.5
+C red      2.0 2.0 2.0
+H yellow   2.0 2.5 2.5
+H yellow   2.0 1.5 1.5
+O octarine 3.0 3.0 3.0
+"""
+with open('myfile.xyzc', 'w') as out:
+    out.write(myfile)
+```
```

Comment:
You can use liquid syntax to have correct syntax highlight preview on a local build

{% highlight python %}
...
{% endhighlight %}